requirements_docs.txt

```diff
@@ -1,7 +1,7 @@
 cartopy==0.21.1
 cmocean==2.0
 colorcet==3.0.1
-geovista==0.2.0
+git+https://github.com/bjlittle/geovista@main
```

Comment:
@tkoyama010 Note that I'm releasing `geovista` 0.3.0 this week and it'll contain the extra functionality that you need for the tutorial.

Did you want me to raise a separate PR or issue to update the dependencies when 0.3.0 is available?

Reply:
Yes. Please. Thanks for your help.

Reply:
No problem. Will do 👍

Reply:
🚀
```suggestion
geovista==0.3.0
```

Reply:
@tkoyama010 You can now update this to be 0.3.0 🥳